### PR TITLE
Work history improvements

### PR DESCRIPTION
**Pull request by frankieroberto**
Adds an entry to describe the recent changes to the Work history section.

Initially I did this as two separate posts (as there were 2 iterations), but it wasn't super clear where to put the "results of the user research" section, and there was also a fair bit of overlap, so @gregknight1 and I decided it made more sense as a single narrative.

### CODE REVIEW

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+related:
+  items:
+  - text: Work history breaks
+    href: https://bat-design-history.netlify.app/apply-for-teacher-training/work-history-breaks/
+    description: Previous post about breaks in work history
+  - text: Work history - August iteration
+    href: https://bat-design-history.netlify.app/apply-for-teacher-training/work-history/
+    description: Previous change to the work history flow
+---
+Discussions with providers revealed that many of them require a “full” work history, rather than just work within the past 5 years, for safeguarding reasons.
```
Comment (gregknight1) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md]: Many providers told us that they want to see a candidate's full work history, not just the last 5 years - for safeguarding reasons.

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+Candidates were spending a significant amount of time filling in the “Skills and experience relevant to teaching” question for each job, treating these as a “mini personal statement”. However providers indicated that they didn’t always read these in detail.
```
Comment (gregknight1) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md]: Comma after 'However,'

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+Analysis of validation errors also revealed that candidates were often leaving the “Skills and experience relevant to teaching” question blank, which then displays an error. Our hypothesis was because they wanted to come back and write this section later.
```
Comment (gregknight1) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md]: Not sure you need to say ...'which then displays an error.' The analysis of validation errors implies this - I can see why you've included it so happy to be swayed to leaving it in.

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+## What we did
+
+The work history section was moved beneath the “Safeguarding” heading on the application review page. This was to try and communicate to candidates that the purpose of the work history was mainly for safeguarding, rather than something used to direct assess them.
```
Comment (gregknight1) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md]: Think we should make this active - _'We moved the work history beneath.._..etc etc'

Typo towards the end - should be 'directly'.

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+We changed the “How long have you been working?” question to “Do you have any work history?” as we no longer needed to guide candidates they only needed to enter their past 5 years of work history.
```
Comment (gregknight1) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md]: Full stop after _'Do you have any work history?_'.  Then _'We didn't need to guide candidates towards their last 5 years of work history any more.'_

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+An answer to this question was added of “No, I have always been in full time education”. This was based on analysis showing that around 20% of candidates are 20 or 21 years old, and have likely gone straight from A-Levels to University, which may explain their lack of work history. Adding this option avoids the need to them to have to explain this in their own words.
```
Comment (gregknight1) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md]: _We added an answer to this question - 'No, I have always.......'_

_Adding this option means they don't have to explain this in their own words_

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+If someone selected “No, I do not have any work history”, then we revealed a textbox asking “Tell us why you’ve been out of the workplace” with some hint text which gave three examples. These were very brief, to indicate to candidates that they did not need to be super detailed.
```
Comment (gregknight1) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md]: Remove _then_ before _we revealed_.

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+
+When someone indicated they did have work history, we took them to a page with an empty table and some guidance that they needed to add their full work history. They could then add a first job.
```
Comment (gregknight1) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md]: ......._an empty table and guidance to add their full work history. They could then add a first job._

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+
+On the Add a job form, we made a number of minor changes and one significant change.
+
+The most significant change was to remove the textbox asking candidates to describe their job and the skills and experience relevant to teaching. This was to reduce the burden on candidates, particularly those with a long work history which may include many jobs done long ago which are completely unrelated to teaching (such as bar or retail work). We did consider making this field optional instead, but considered that many candidate would feel they _ought_ to write something anyway, which likely wouldn’t get read or be helpful.
```
Comment (gregknight1) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md]: _........with long work histories which may include many jobs done long ago that are unrelated to teaching - like bar or retail work._

Last sentence _candidate_ needs pluralising

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+
+Minor changes included:
+
+* Removing the working pattern question for part time jobs – as this didn’t seem to be strongly required, and would be a complicated to answer for part time or casual jobs done a long time ago..
```
Comment (gregknight1) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md]: ditch the _as_

2 full stops for a reason?

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+* Removing the hint text suggesting candidates add the name of their own company if self-employed, as this didn’t seem to be required.
```
Comment (gregknight1) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md]: follow the same pattern used in the first bullet:

- _........if self-employed - this didn't seem to be required._

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+* Changing the end date from “End date (leave blank if this is your current role)” to “Are you still working in this job?”. If the candidate answered “No” then a ”When did you leave this job?” question was revealed.
+* Changing the start date from “Start date” to “When did you start this job?”, for consistency with the “When did you leave this job?” question, and to better reflect the fact that we don’t ask for a full date, only month and year.
+
+Upon adding a job, candidates were returned to the Work history review page, rather than being asked whether or not they wanted to add another job. This was partly to solve an issue seen in production where candidates go to add another job, but then leave all the fields blank and submit the form, causing validation errors. Our hypothesis was that this is because they either changed their mind or didn’t realise that they would need to add another job immediately. Additionally, by taking candidates back to the review page after adding each job, they are able to review their progress in adding jobs so far, and can remind themselves where they’ve got to – which may be more important when adding lots and lots of jobs.
```
Comment (gregknight1) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md]: _When candidates added a job, they were returned to......._

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+
+The review page was redesigned. Instead of using summary cards for each job, the jobs were presented in a table format. This was intended to remove the duplicatation caused by repeating the field names in each summary card, and make it easier for candidates to scan the page, as each job takes up less vertical height.
```
Comment (gregknight1) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md]: _We redesigned the review page._

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md [hunk 14]
```diff
@@ -0,0 +1,76 @@
...
+
+Pull request: https://github.com/DFE-Digital/apply-for-teacher-training-prototype/pull/434
+
+## Results of the user research
+
+We tested a prototype with XX candidates. We recruited older candidates so that we could test the new section with people who likely had a longer work history.
+
+We also tested a mock-up of applications showing long work histories with providers, in order to check our assumptions.
```
Comment (gregknight1) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md]: Remove _in order_ 

_......with _providers, to check our assumptions.__

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md [hunk 15]
```diff
@@ -0,0 +1,76 @@
...
+
+Candidates were able to add lots of jobs fairly quickly - likely much more quickly than if they were still being asked for skills and experience relevant to teaching for each job.
+
+Some candidates were unclear that they needed to add their complete work history, either not reading or ignoring the guidance. Instead they only added 'relevant' or 'proper' jobs, and omitting their first jobs working in bars or in retail.
+
+Some candidates experessed surprise that they weren’t asked for more details. Whilst some were happy with how quick it was, others wanted to be able to explain more about their roles to support their application.
+
+Some candidates hesitated when it came to entering the start and end year and months for some of their early roles, and had to spend some time thinking about this. When asked, they suggested that they weren’t completely certain, but were able to either work it out or enter a reasonable guess by anchoring their early jobs to other significant events in their lives.
```
Comment (gregknight1) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md]: take the s off _months_

File: app/posts/apply-for-teacher-training/2021-02-10-work-history-iteration-post-testing.md [hunk 16]
```diff
@@ -0,0 +1,40 @@
...
+  - text: Work history iteration for testing
+    href: https://bat-design-history.netlify.app/apply-for-teacher-training/work-history-iteration-for-testing/
+    description: Changes to the work history flow made for testing
+---
+
+We made some further changes to the Work history section in response to user testing.
+
+## What we did
+
+We moved the section back underneath a Work experience section, and reinstated the Unpaid experience section.
```
Comment (gregknight1) [inline app/posts/apply-for-teacher-training/2021-02-10-work-history-iteration-post-testing.md]: Think the _a_ after _underneath_ should be _the_

File: app/posts/apply-for-teacher-training/2021-02-10-work-history-iteration-post-testing.md [hunk 17]
```diff
@@ -0,0 +1,40 @@
...
+
+We reordered the Add job form, to make the ‘Name of employer’ the first field, as this was often what candidates recalled first.
+
+We renamed ‘Job title’ to ‘Role’, as candidates may not remember their exact job title (or even have had one), and so ‘Role’ implies they can use a short description instead.
+
+We added some hint text to ‘Role’ saying “If you had more than one role with this employer, enter your last role.” as feedback from providers suggested that listing out each promotion within a employment was unnecessary.
+
+We added a checkbox labelled “I’m not sure of the exact month or year I started” and ”I’m not sure of the exact month or year I left” so that candidates can use these when they could not recall the relevant months.
```
Comment (gregknight1) [inline app/posts/apply-for-teacher-training/2021-02-10-work-history-iteration-post-testing.md]: _.......or year I left' for candidates who struggled with the exact months and years._

File: app/posts/apply-for-teacher-training/2021-02-10-work-history-iteration-post-testing.md [hunk 18]
```diff
@@ -0,0 +1,40 @@
...
+
+We renamed “Did this job involve working in a school or with childen?” to “Did you use skills relevant to teaching in this job?” as providers indicated that they were most interested in _relevant_ work history, which could encompass a wider variety of jobs.
+
+We added the hint text “You can go into more detail about this in your personal statement.” to the ”Did this job involve working in a school or with childen?” question to remind candidates that they can give more details about relevant jobs in their personal statement.
```
Comment (gregknight1) [inline app/posts/apply-for-teacher-training/2021-02-10-work-history-iteration-post-testing.md]: _.....question to encourage candidates to give more details about relevant jobs in their personal statement._

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md (hunk 3 above)
Comment (frankieroberto) [reply app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md]: Agree, let's kill it.

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md (hunk 10 above)
Comment (frankieroberto) [reply app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md]: Opps, well spotted.

File: app/posts/apply-for-teacher-training/2021-02-10-work-history-iteration-post-testing.md (hunk 16 above)
Comment (frankieroberto) [reply app/posts/apply-for-teacher-training/2021-02-10-work-history-iteration-post-testing.md]: 👍

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md [hunk 19]
```diff
@@ -0,0 +1,146 @@
...
+
+Our hypothesis was that this is because they either changed their mind or didn’t realise that they would need to add another job immediately. Additionally, by taking candidates back to the review page after adding each job, they are able to review their progress in adding jobs so far, and can remind themselves where they’ve got to – which may be more important when adding lots and lots of jobs.
+
+We redesigned the review page. Instead of using summary cards for each job, the jobs were presented in a table format. This was intended to remove the duplication caused by repeating the field names in each summary card, and make it easier for candidates to scan the page, as each job takes up less vertical height.
+
+## Results of the user research
+
+We tested a prototype with XX candidates. We recruited older candidates so that we could test the new section with people who likely had a longer work history.
```
Comment (frankieroberto) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md]: Need to replace XX with the actual number.

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md [hunk 20]
```diff
@@ -0,0 +1,146 @@
...
+related:
+  items:
+  - text: Work history breaks
+    href: https://bat-design-history.netlify.app/apply-for-teacher-training/work-history-breaks/
+    description: Previous post about breaks in work history
+  - text: Work history - August iteration
+    href: https://bat-design-history.netlify.app/apply-for-teacher-training/work-history/
+    description: Previous change to the work history flow
+---
+Many providers told us that they want to see a candidate's full work history, not just the last 5 years - for safeguarding reasons.
```
Comment (adamsilver) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md]: Can you elaborate what ‘safeguading reasons’ means?

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md [hunk 21]
```diff
@@ -0,0 +1,146 @@
...
+
+Candidates were spending a significant amount of time filling in the “Skills and experience relevant to teaching” question for each job, treating these as a “mini personal statement”. However, providers indicated that they didn’t always read these in detail.
```
Comment (adamsilver) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md]: Providers don't read them BECAUSE they aren't always relevant to teaching. Worth saying I think.

**Comment by adamsilver** [comment]
There is no screenshot for ‘We moved the work history section beneath the “Safeguarding” heading on the application review page.’

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md [hunk 22]
```diff
@@ -0,0 +1,146 @@
...
+
+## What we did
+
+We moved the work history section beneath the “Safeguarding” heading on the application review page. This was to try and communicate to candidates that the purpose of the work history was mainly for safeguarding, rather than something used to directly assess them.
+
+We changed the “How long have you been working?” question to “Do you have any work history?”. We didn't need to guide candidates towards entering only their last 5 years of work history anymore.
+
+We added an answer to this question – “No, I have always been in full time education”. This was based on analysis showing that around 20% of candidates are 20 or 21 years old, and have likely gone straight from A-Levels to University, which may explain their lack of work history. Adding this option means that they don't have to explain this in their own words.
+
+If someone selected “No, I do not have any work history”, we revealed a textbox asking “Tell us why you’ve been out of the workplace” with some hint text which gave three examples. These were very brief, to indicate to candidates that they did not need to be super detailed.
```
Comment (adamsilver) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md]: I am not sure why you're saying this? Did you change this in any way?

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md [hunk 23]
```diff
@@ -0,0 +1,146 @@
...
+
+On the Add a job form, we made a number of minor changes and one significant change.
+
+The most significant change was to remove the textbox asking candidates to describe their job and the skills and experience relevant to teaching. This was to reduce the burden on candidates, particularly those with long work histories which may include many jobs done long ago that are unrelated to teaching - like bar or retail work. We did consider making this field optional instead, but considered that many candidates would feel they _ought_ to write something anyway, which likely wouldn’t get read or be helpful.
+
+Minor changes included:
+
+* Removing the working pattern question for part time jobs – this didn’t seem to be strongly required, and would be complicated to answer for part time or casual jobs done a long time ago.
+* Removing the hint text suggesting candidates add the name of their own company if self-employed – this didn’t seem to be required.
+* Changing the end date from “End date (leave blank if this is your current role)” to “Are you still working in this job?”. If the candidate answered “No” then a ”When did you leave this job?” question was revealed.
```
Comment (adamsilver) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md]: I like this change, but can you explain why you did it?

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md [hunk 24]
```diff
@@ -0,0 +1,146 @@
...
+
+When a candidate added a job, they were returned to the Work history review page, rather than being asked whether or not they wanted to add another job. This was partly to solve an issue seen in production where candidates go to add another job, but then leave all the fields blank and submit the form, causing validation errors.
+
+Our hypothesis was that this is because they either changed their mind or didn’t realise that they would need to add another job immediately. Additionally, by taking candidates back to the review page after adding each job, they are able to review their progress in adding jobs so far, and can remind themselves where they’ve got to – which may be more important when adding lots and lots of jobs.
+
+We redesigned the review page. Instead of using summary cards for each job, the jobs were presented in a table format. This was intended to remove the duplication caused by repeating the field names in each summary card, and make it easier for candidates to scan the page, as each job takes up less vertical height.
```
Comment (adamsilver) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md:50]: Can you link ‘summary card’ to the an old design post that shows the pattern?

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md [hunk 25]
```diff
@@ -0,0 +1,146 @@
...
+
+## Results of the user research
+
+We tested a prototype with XX candidates. We recruited older candidates so that we could test the new section with people who likely had a longer work history.
+
+We also tested a mock-up of applications showing long work histories with providers, to check our assumptions.
```
Comment (adamsilver) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md:56]: Can you link to this or show this? Right now I have to imagine it.

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md [hunk 26]
```diff
@@ -0,0 +1,146 @@
...
+
+We found that moving the work history section beneath the Safeguarding heading did not communicate to candidates why we were asking for this information. If anything, they assumed that we might be asking for experience of safeguarding within their work (in part because many of them had some awareness of safeguarding from working in schools or with children).
+
+Candidates were able to add lots of jobs fairly quickly - likely much more quickly than if they were still being asked for skills and experience relevant to teaching for each job.
+
+Some candidates were unclear that they needed to add their complete work history, either not reading or ignoring the guidance. Instead they only added 'relevant' or 'proper' jobs, and omitting their first jobs working in bars or in retail.
+
+Some candidates expressed surprise that they weren’t asked for more details. Whilst some were happy with how quick it was, others wanted to be able to explain more about their roles to support their application.
```
Comment (adamsilver) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md:64]: You say ‘some’ a lot. Would be good to get numbers out of total numbers here and all the other places where you said ‘some’

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md [hunk 27]
```diff
@@ -0,0 +1,146 @@
...
+
+Some candidates hesitated when it came to entering the start and end year and month for some of their early roles, and had to spend some time thinking about this. When asked, they suggested that they weren’t completely certain, but were able to either work it out or enter a reasonable guess by anchoring their early jobs to other significant events in their lives.
+
+One candidate consistently entered their employers’ names within the Job title field, and then had to correct themselves. Other candidates entered the employer names first, and then returned to the job title field later having had to give it more thought.
+
+Some candidates suggested that they would be able to get the relevant information from their CV, but no candidates had a CV that included their complete work history.
+
+Not all candidates responded to the prompts to enter explanations of breaks within their work history. Those that did were able to explain their break, eg by mentioning unemployment or raising children, with no stress and using only a brief explanation.
```
Comment (adamsilver) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md]: ‘eg’ > ‘for example’

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md [hunk 28]
```diff
@@ -0,0 +1,146 @@
...
+
+## What we changed
+
+We moved the section back underneath the Work experience section, and reinstated the Unpaid experience section.
+
+We reordered the Add job form, to make the ‘Name of employer’ the first field, as this was often what candidates recalled first.
+
+We renamed ‘Job title’ to ‘Role’, as candidates may not remember their exact job title (or even have had one), and so ‘Role’ implies they can use a short description instead.
+
+We added some hint text to ‘Role’ saying “If you had more than one role with this employer, enter your last role.” as feedback from providers suggested that listing out each promotion within a employment was unnecessary.
```
Comment (adamsilver) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md]:
```suggestion
We added hint text to ‘Role’ saying “If you had more than one role with this employer, enter your last role.” as feedback from providers suggested that listing out each promotion within a employment was unnecessary.
```

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md [hunk 29]
```diff
@@ -0,0 +1,146 @@
...
+
+We added a checkbox labelled “I’m not sure of the exact month or year I started” and ”I’m not sure of the exact month or year I left” for candidates who struggled with the exact months and years.
+
+We renamed “Did this job involve working in a school or with children?” to “Did you use skills relevant to teaching in this job?” as providers indicated that they were most interested in _relevant_ work history, which could encompass a wider variety of jobs.
```
Comment (adamsilver) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md]: Do not use italics.

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md [hunk 30]
```diff
@@ -0,0 +1,146 @@
...
+
+We added the hint text “You can go into more detail about this in your personal statement.” to the ”Did this job involve working in a school or with children?” question to encourage candidates to give more details about relevant jobs in their personal statement.
+
+On the review page, we simplified the design, moving away from a table and just using headings with horizontal lines between jobs, enclosed within a bordered section. This removed the need to include column headings and a table caption, and addressed accessibility issues with having used a row of merged cells for the break in history prompts.
```
Comment (adamsilver) [inline app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md]:
```suggestion
On the review page, we moved away from a table and just using headings with horizontal lines between jobs, enclosed within a bordered section. This removed the need to include column headings and a table caption, and addressed accessibility issues with having used a row of merged cells for the break in history prompts.
```

Comment (frankieroberto) [reply app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md]: Good call, although you made the rest of the sentence ungrammatical... I think this would work (also not sure about 'moved' any more):

> On the review page, we switched from a table to headings with horizontal lines between jobs, enclosed within a bordered section.

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md (hunk 29 above)
Comment (frankieroberto) [reply app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md]: You're an italic.

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md (hunk 22 above)
Comment (frankieroberto) [reply app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md]: Yes, previously the textbox was on a follow-up page rather than conditionally-revealed.

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md (hunk 26 above)
Comment (frankieroberto) [reply app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md:64]: Good point - I'm hoping the research team can help flesh out this section a bit more.

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md (hunk 21 above)
Comment (frankieroberto) [reply app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md]: I'm not sure that necessarily the only reason - sometimes they might overlap with the personal statement, or just be too long-winded and vague. I'll see if I can find some more information from the original research conversations.

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md (hunk 22 above)
Comment (frankieroberto) [reply app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md]: And the hints were new. I'll see if I can make this clearer.

**Comment by frankieroberto** [comment]
[Design history entry for work history](https://trello.com/c/286VwAbS/3064-design-history-entry-for-work-history)

File: app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md (hunk 26 above)
Comment (mayawise) [reply app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md:64]: Numbers are relevant when you're estimating how prevalent an issue is in a user population –– they're not so useful to  with very small sample because they usually tell us very little about how often it would actually happen in real life. 

The important point is that we discovered and fixed issues before real users had to deal with them! (Not how many in the small sample reported that issue.)
